package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "chesster",
+  "version": "1.0.0",
+  "description": "Bot for Lichess4545 Slack group",
+  "main": "chesster.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "async": "^2.0.0-rc.3",
+    "botkit": "^0.1.1",
+    "chai": "^3.5.0",
+    "fast-levenshtein": "^1.1.3",
+    "google-spreadsheet": "^2.0.1",
+    "moment": "^2.13.0",
+    "underscore": "^1.8.3"
+  },
+  "devDependencies": {
+    "mocha": "^2.4.5"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aweidner/Chesster.git"
```

Comment:
Yeah correct, and the version should be whatever the current version of Chesster is.  I'm not sure, do we have it somewhere?